### DISCUSSION TITLE
Feature Request: support diskEncryptionKey/KmsKey for google batch executor

### DISCUSSION THREAD

**Discussion by livanaz** [nextflow-io/nextflow#5262] [Ideas]
Hi Nextflow Team, 

Google batch and Compute Engine can use disks that support a customer managed encryption key(https://cloud.google.com/kms/docs/cmek). Is it possible to add this feature to nextflow? 

Thank you!

Best,

**Comment by pditommaso** [comment]
Feel free to provide a pull request for it.
